### DISCUSSION TITLE
Mobile Preview affecting Destkop view

### DISCUSSION THREAD

**Discussion by joshgk00** [vanjarosoftware/Vanjaro.Platform#1964] [Q&A] [answered]
I may be mistaken, but in Vanjaro, couldn't you change settings/layout specific to a mobile layout? For example a grid with 2 columns, first column has text block the 2nd has an image block. If on Mobile Portrait I want to reverse the column order. I thought that I could switch to the mobile layout and make the change to the grid and then publish the page and have it only affect the mobile layout, not the desktop layout. When I try that now, it also changes the Desktop mode as well. Is this functionality supported?

**Comment by meetmandeep** [comment, marked as answer]
Yes. Take a look at this video: https://www.youtube.com/watch?v=Z2q0JqoxS40

**Reply by joshgk00** [reply]
Watched the video and followed the steps but what I'm seeing is the changes are not isolated to the viewport I'm previewing. Mobile changes are showing in desktop version. Possible bug?

**Reply by joshgk00** [reply]
I tested this on another site running Vanjaro 1.2.0 as an extension and the functionality isn't working there either. I believe there's a bug.

**Reply by joshgk00** [reply]
Here's a quick reproduction of the issue
https://www.screencast.com/t/HuX59hrmTVB

**Reply by Gaganbird** [reply]
You are not doing it correctly. Please follow the below steps:

1. Select Tablet
2. Select Grid
3. Click Styling tab > Responsive
4. Reverse Columns > Yes
5. Publish
